modules/installation-osp-setting-cloud-provider-options.adoc

```diff
@@ -31,7 +31,7 @@ $ vi openshift/manifests/cloud-provider-config.yaml
 
 . Modify the options based on link:https://v1-18.docs.kubernetes.io/docs/concepts/cluster-administration/cloud-providers/#cloud-conf[the `cloud.conf` specification].
 +
-Configuring Octavia for load balancing is a common case. For example:
+Configuring Octavia for load balancing is a common case for installations that do not use Kuryr. For example:
```

Comment:
@MaysaMacedo Actually, could one of these work better?

"installations" -> "clusters"
"installations" -> "deployments" 

After that, I think this just needs a QE ack.

Reply:
clusters I think is better. Let me update it